### ISSUE TITLE
Add more sensors from the API

### ISSUE THREAD

**Issue by Amateur-God** [DurgNomis-drol/ha_toyota#315]
Could you add more sensors from the api such as battery level, vehicle status, vehicle image,, charging status, notifications etc or could you provide a link to the api docs as i couldnt find anything 

i see most of these are available in your 
https://github.com/DurgNomis-drol/mytoyota/blob/master/mytoyota/models/endpoints/electric.py

but i cant see them in the home assistant integration

**Comment by CM000n** [comment]
Closed due to discontinuation of the project. See: https://github.com/DurgNomis-drol/ha_toyota/issues/346

**Comment by Amateur-God** [comment]
@CM000n do I need to open an issue/request on the new repo? 

Happy to submit a PR for this myself to the new repo if you have any links to API docs for Toyota API

**Comment by ajain-93** [comment]
Yes, needs to reopen in the new repo if feature is still requested.
Also, there is are no API docs available for the MyToyota API, most of development here is by various people reverse-engineering the endpoints used by the app.
